CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ else()
       )
     endif()
 
+    if(AMDGPU_TARGETS AND NOT GPU_TARGETS)
+      message( DEPRECATION "AMDGPU_TARGETS use is deprecated. Use GPU_TARGETS." )
+    endif()
+
     if (AMDGPU_TARGETS)
```

Comment:
Should modify here as well.

Reply:
The intention is that AMDGPU_TARGETS should still work, it's just marked as deprecated, so I don't want to change this block. Additionally, we're trying to move away from "all" as an accepted target (as it doesn't convey the same thing across all components, and what "all" means is not clearly defined), so I don't want to update that processing.